From a832764e7300d48468a6812077b899a4beed271b Mon Sep 17 00:00:00 2001
From: Masamichi Hosoda <trueroad@trueroad.jp>
Date: Sat, 22 Aug 2015 21:20:17 +0900
Subject: [PATCH] Fix fontforge version detection

Newer fontforge shows git hash in `fontforge --version`.
In that case, configure script
might recognize it as date (version).
---
 aclocal.m4 | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/aclocal.m4 b/aclocal.m4
index 48d0b77..0037c58 100644
--- a/aclocal.m4
+++ b/aclocal.m4
@@ -42,7 +42,8 @@ AC_DEFUN(STEPMAKE_GET_VERSION, [
 
     if test -z "$_ver"; then
         ## If empty, try date [fontforge]
-        eval _ver=\"\`("$1" --version || "$1" -V) 2>&1 | grep '[0-9]\{6,8\}' \
+        eval _ver=\"\`("$1" --version || "$1" -V) 2>&1 \
+	    | grep '\(^\|[^0-9a-f]\)[0-9]\{6,8\}\([^0-9a-f]\|$\)' \
 	    | head -n 1 \
 	    | sed -e 's/^[^.0-9]*//' -e 's/[^.0-9]*$//'\`\"
     fi
-- 
2.6.2

